pkg/deploy/dev-workspace/dev_workspace_test.go

```diff
@@ -62,6 +67,19 @@ func TestReconcileDevWorkspace(t *testing.T) {
 	if !done {
 		t.Fatalf("Dev Workspace operator has not been provisioned")
 	}
+
+	t.Run("defaultCheManagerDeployed", func(t *testing.T) {
```

Comment:
Besides this it would be nice to have some checks on e2e tests like here.
https://github.com/eclipse/che-operator/blob/master/.ci/oci-nightly-olm.sh#L43-L49